spec/app/lib/bing_api_v7_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+require 'json'
+require 'webmock/rspec'
+require 'support/vcr_setup'
```

Comment:
just uncomment `Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }` in your `spec/rails_helper.rb`